content/docs/client-options.md

```diff
@@ -41,6 +41,7 @@ These clients are compatible with our [staging endpoint for ACME v2](https://com
 - [ACME Tiny](https://github.com/diafygi/acme-tiny)
 - [itr-acme-client PHP library](https://github.com/ITronic/itr-acme-client)
 - [acmebot](https://github.com/plinss/acmebot)
+- [SSL for free](https://www.sslforfree.com/) (Fully in-browser process, inc. CSR generation, wildcard certificate available)
```

Comment:
For the ACME v2 section we've been listing clients without extra explanatory text in parens. Can you remove the `"(Fully in browser process, ...)"` text?